packages/fxa-content-server/app/scripts/views/settings/subscription.js

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+import Cocktail from 'cocktail';
+import FormView from '../form';
+import SettingsPanelMixin from '../mixins/settings-panel-mixin';
+import Template from 'templates/settings/subscription.mustache';
+
+const View = FormView.extend({
+  template: Template,
```

Comment:
I know parts of this module are going to be changed, but can you add a unit test suite to ensure that it renders without blowing up?